pyproject.toml

```diff
@@ -3,8 +3,8 @@
 
 [build-system]
 requires = [
-  "setuptools == 68.1.0",
-  "setuptools_scm[toml] == 7.1.0",
+  "setuptools == 75.5.0",
+  "setuptools_scm[toml] == 8.1.0",
   "frequenz-repo-config[lib] == 0.10.0",
```

Comment:
what about this version?

Reply:
Doesn't this need updating?

Reply:
It is updating according to the repo config specs. `75.5.0` and `8.1.0` are the versions required.

Reply:
This is also in PR #119 and #120 from Dependabot.

Reply:
I mean why is repo config not updated here?

Reply:
Sorry, I thought you meant the setup tools update... Will check!